data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report2_row_suppression.py

```diff
@@ -76,8 +76,8 @@
 df1
 
 # # 2 Verify all fields identified for suppression in the OBSERVATION table have been removed from the table in the deid dataset.
-
-# ## error in new cdr
+#
+# If there are issues view them in more detail using query 2.1
```

Comment:
Updated these descriptions. The second query in several checks would be helpful for any failure investigations.